grakn/rpc/session.py

```diff
@@ -70,9 +75,11 @@ def _transmit_pulse(self):
             return
         pulse_req = session_proto.Session.Pulse.Req()
         pulse_req.session_id = self._session_id
-        is_alive = self._grpc_stub.session_pulse(pulse_req).is_alive
-        if is_alive:
-            self._pulse = self._scheduler.enter(5, 1, self._transmit_pulse, ())
+        res = self._grpc_stub.session_pulse(pulse_req)
+        if res.alive:
```

Comment:
Fixing the pulse scheduler also exposed another bug: the field was named wrongly (was `is_alive`, should be `alive`)